website/docs/r/acm_certificate_validation.html.markdown

```diff
@@ -133,6 +133,12 @@ The following arguments are supported:
 * `certificate_arn` - (Required) The ARN of the certificate that is being validated.
 * `validation_record_fqdns` - (Optional) List of FQDNs that implement the validation. Only valid for DNS validation method ACM certificates. If this is set, the resource can implement additional sanity checks and has an explicit dependency on the resource that is implementing the validation
 
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ARN of the certificate validated
```

Comment:
Hi @gmembre-zenika 👋 It looks like the code itself actually saves the `id` attribute as the [issued at timestamp](https://docs.aws.amazon.com/acm/latest/APIReference/API_CertificateDetail.html#ACM-Type-CertificateDetail-IssuedAt):

https://github.com/terraform-providers/terraform-provider-aws/blob/8847f0cb3e1cebd67cb2d5ffe7b45c059a271366/aws/resource_aws_acm_certificate_validation.go#L156

Were you seeing different behavior? If not, we should update this line to include the correct information from the API reference:

```suggestion
* `id` - The time at which the certificate was issued
```

Reply:
Hi, you are correct, I miss-read the code 👍